website/content/docs/configure.mdx

```diff
@@ -1,87 +1,73 @@
+| Windows | `%APPDATA%\packer.d\` | |
+
+
+### Examples
+
+- **Unix**: If `$PACKER_CONFIG_DIR` is set to `/home/packer`, then the configuration directory is `/home/packer/.packer.d/`. Packer does not check other values.
+- **Windows**: If `PACKER_CONFIG_DIR` is set to `C:/`, then the configuration directory is `C:/packer.d/`. Packer does not check other values.
 
-- On a Unix system, if the `$PACKER_CONFIG_DIR` environment variable is set to
-  `/home/packer`, the config directory will be: `/home/packer/.packer.d/` and
-  other values will not be checked.
-- On a Windows system, if the `PACKER_CONFIG_DIR` environment variable is set to `C:/`,the
-  config directory will be: `C:/packer.d/` and other values will not be checked.
 
 <a id="packer-s-config-file"></a>
 
-## Packer's config file (deprecated)
+##  Install a JSON configuration file
+
+This installation method is deprecated since 1.7. Instead, use the [HCL2
+`required_plugins` block](/packer/docs/templates/hcl_templates/blocks/packer#specifying-plugin-requirements) to configure how Packer installs plugins. 
 
-Packer can optionally read a JSON file for the end user to set core settings.
-The config file of Packer will be looked up on the following paths:
+You use a JSON file to set core Packer settings. Packer checks the following paths for the configuration file:
 
 | Unix                             | Windows                           |
 | -------------------------------- | --------------------------------- |
 | `${PACKER_CONFIG}`               | `%PACKER_CONFIG%`                 |
 | `${HOME}/.packerconfig`          | `%APPDATA%\packer.config\`        |
 
-The format of the configuration file is basic JSON.
 
-### Packer config file configuration Reference
+### JSON configuration file reference 
+
+Use basic JSON to configure the file. You can specify the configuration parameters for the core
+configuration file. All parameters are optional. None of these are required, since all have defaults.
 
-Below is the list of all available configuration parameters for the core
-configuration file. None of these are required, since all have defaults.
+- `plugin_min_port`: Number that specifies the lowest port that Packer can use for communicating with plugins. Packer communicates with plugins over TCP connections on your local Unix host. Default is `10000`. We recommend setting a wide range between `plugin_min_port` and `plugin_max_port` so that Packer has access to at least 25 ports on a single run.
```

Comment:
```suggestion
- `plugin_min_port`: Number that specifies the lowest port that Packer can use for communicating with plugins. Packer communicates with plugins over TCP or Unix sockets on your local host. Default is `10000`. We recommend setting a wide range between `plugin_min_port` and `plugin_max_port` so that Packer has access to at least 25 ports on a single run.
```
Ah, OK. Thanks for clarifying.